tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -311,8 +311,12 @@ namespace vz_projector {
     }
 
     private getLabelFromIndex(pointIndex: number): string {
-      const point = this.projector.dataSet.points[pointIndex];
-      return point.metadata[this.selectedMetadataField].toString();
+      const metadata = this.projector.dataSet.points[pointIndex].metadata[
+        this.selectedMetadataField
+      ];
+      return metadata !== undefined
+        ? metadata.toString()
+        : `unknown #${pointIndex}`;
```

Comment:
Stupid question, is there a reason why this is lower case (as opposed to one in PlotAdapter)? Is this good for UX?